### DISCUSSION TITLE
Train "text encoder" only at beginning and end of training?

### DISCUSSION THREAD

**Discussion by Jonseed** [d8ahazard/sd_dreambooth_extension#184] [Q&A]
Someone said that it is better to train the text encoder only at beginning (like the first 1k steps), and at the end (like the last 500 steps), but not during the rest of the training. Their reasoning was that training the text encoder during the whole time will overfit the model with your subject/style, and it will appear everywhere. They said you just need to fine-tune the text encoder just enough to get the data from your images, but then spend the rest of the training on the UNet alone.

Is this common? Does this make sense?

**Comment by smnnekho** [comment]
i think this is LastBen's approach (https://github.com/TheLastBen/fast-stable-diffusion)

you can actually just test this by starting Training with text encoder for x steps, continue without it and do some final steps with it activated again
